library/src/main/assets/google_map.html

```diff
@@ -90,6 +92,26 @@
       icon: 'http://maps.google.com/mapfiles/ms/icons/red-dot.png'
   });
 
+  if (title == "null" && snippet == "null") {
+        <!-- Title and snippet not set, don't create default InfoWindow -->
+  } else if (title != "null" && snippet == "null") {
```

Comment:
@jvartanian94 could we wrap the entire block in a check to ensure that at least one of them in not null.